packages/@aws-cdk/aws-stepfunctions/lib/step-functions-task.ts

```diff
@@ -78,3 +78,33 @@ export interface StepFunctionsTaskConfig {
    */
   readonly metricDimensions?: cloudwatch.DimensionHash;
 }
+
+/**
+ * Ways to call an integrated AWS service in Step Functions
+ *
+ * @default FIRE_AND_FORGET
+ *
+ * @see
+ * https://docs.aws.amazon.com/step-functions/latest/dg/connect-to-resource.html
+ */
+export enum ServiceIntegrationPattern {
+  /**
+   * Call a service and progress to the next state immediately after the API call completes
+   */
+  FIRE_AND_FORGET = 'FIRE_AND_FORGET',
+
+  /**
+   * Call a service and wait for a job to complete.
+   */
+  SYNC = 'SYNC',
+
+  /**
+   * Call a service with a task token and wait until that token is returned by SendTaskSuccess/SendTaskFailure with paylaod
+   */
+  WAIT_FOR_TASK_TOKEN = 'WAIT_FOR_TASK_TOKEN'
+}
+
+export const resourceArnSuffix = new Map<ServiceIntegrationPattern, string>();
```

Comment:
I'd rather this map lives in the `aws-stepfunctions-tasks` library (NOT exported from `index.ts`).

Reply:
Thank you for the quick reply. I will move this map to `aws-stepfunctions-tasks`.